package.json

```diff
@@ -116,7 +116,7 @@
     "tiny-queue": "^0.2.1",
     "uglifyjs-webpack-plugin": "^1.2.7",
     "uuid": "^3.1.0",
-    "uws": "^10.148.1",
+    "uws": "10.148.0",
```

Comment:
uws v10.148.1 does not support Alpine.

Reply:
The source code of uws v10.148.1 is not published on GitHub．

https://github.com/uNetworking/uWebSockets-bindings/blob/95e44dba351ed8e6d91aa559d4bb9bb539e1c681/nodejs/dist/package.json#L3

🤔